File: aoj/volume100/csharp/A10010.cs
using System;

class AOJ
{
	static void Main()
	{
		while (true)
		{
			var input = Console.ReadLine().Split();
			var a = int.Parse(input[0]);
			var b = int.Parse(input[2]);
			var op = input[1];
			if (op == "?")
				break;

			Console.WriteLine(Calc(op, a, b));
		}
	}

	static int Calc(string op, int a, int b)
	{
		switch (op) {
		case "+": return a + b;
		case "-": return a - b;
		case "*": return a * b;
		case "/": return a / b;
		}
		return 0;
	}
}

File: aoj/volume100/c/A10010.c
#include <stdio.h>

int calc(char op, int a, int b) {
	switch (op) {
	case '+': return a + b;
	case '-': return a - b;
	case '*': return a * b;
	case '/': return a / b;
	default : return 0;
	}
}

int main() {
	int a, b;
	char op;
	for (;;) {
		scanf("%d %c %d", &a, &op, &b);
		if (op == '?')
			break;

		printf("%d\n", calc(op, a, b));
	}
	return 0;
}
